### PR TITLE
Adds HyPhy version flag to command line options

### PR DESCRIPTION
**Pull request by sjspielman**
Added -v/-V flag to reveal version
Renamed LEISR output to <name>LEISR.json for consistency

Tagging @spond for `unix.cpp` review.

### CODE REVIEW

File: src/mains/unix.cpp [hunk 1]
```diff
@@ -525,6 +530,12 @@ void    ProcessConfigStr (_String& conf)
             exit (0);
         }
 
+        case 'v':
+        case 'V': {
+            fprintf(stderr, "\n%s\n", hy_version );
```
Comment (spond) [inline src/mains/unix.cpp]: I would just use

```
fprintf(stderr, "\n%s\n", GetVersionString()->getStr() );
```

that way you don't have to create the version string regradless of what the options are.

File: src/mains/unix.cpp [hunk 2]
```diff
@@ -517,6 +520,8 @@ long    DisplayListOfPostChoices (void)
 void    ProcessConfigStr (_String& conf)
 {
     _String errMsg;
+    _String  verString (GetVersionString().getStr());
```
Comment (spond) [inline src/mains/unix.cpp]: Delete these two lines when you make the change below.

Comment (sjspielman) [reply src/mains/unix.cpp]: Having attempted this change, here is what the compiler has to say:
```
/Users/sjspielman/hyphy/src/mains/unix.cpp:536:57: error: base operand of '->' has non-pointer type '_String'
             fprintf(stderr, "\n%s\n", GetVersionString()->getStr() );
                                                         ^~
make[3]: *** [CMakeFiles/HYPHYMP.dir/src/mains/unix.cpp.o] Error 1
```

Comment (spond) [reply src/mains/unix.cpp]: Oh yeah, do `GetVersionString().getStr()`

**Pull request merged**